### DISCUSSION TITLE
Load eslint-plugins similar to parsers

### DISCUSSION THREAD

**Discussion by michael-siek** [eslint/eslint#14917] [Q&A]
I was wondering if it would be possible to be able to load eslint-plugins similar to parsers
```js
plugins: [require.resolve('some-plugin')]
```

**Comment by nzakas** [comment]
No it’s not.

**Reply by michael-siek** [reply]
Is it by chance on the roadmap to be able to do so?

**Reply by nzakas** [reply]
Not for the current config system. In the new config system you will be able to load from wherever you want.

**Reply by michael-siek** [reply]
Do you by chance know when the new config system might be released?

**Reply by nzakas** [reply]
No. It will all depend on when v8.0.0 is finalized and available time. You can follow this issue for updates: https://github.com/eslint/eslint/issues/13481
